bonsai/Modules/Onboarding/Onboarding.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  Onboarding.swift
+//  bonsai
+//
+//  Created by antuan.khoanh on 06/08/2022.
+//
+
+import SwiftUI
+
+struct OnboardingDataSource {
+    let image: String
+    let title: String
+    let description: String
+}
+
+struct Onboarding: View {
+    
+    @State private var selection = 0
+    
+    private let dataSource: [OnboardingDataSource] = [
+        .init(
+            image: "bonsai_1",
+            title: "Why do we use it?",
+            description: "Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis enim velit mollit. Exercitation ve"),
+        .init(
+            image: "bonsai_2",
+            title: "Where does it come from?",
+            description: "Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. "),
+        .init(
+            image: "bonsai_3",
+            title: "Where can I get some?",
+            description: "There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected humour, or randomised words which don't look even slightly believable."),
+    ]
+    
+    var body: some View {
+        VStack {
+            ZStack {
+                
+                if selection == 0 {
```

Comment:
Why did u do in such why? Isn't it easier to do just 
`OnboardingCard(dataSource: dataSource[selection])` and check if selection isn't greater that 2

Reply:
Because of SwiftUI bugs. Trust me, we tried x) But this workaround seems needed